Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2255,7 +2255,18 @@ extension BrowserViewController: WKNavigationDelegate {
             // forward/backward. Strange, but LayoutChanged fixes that.
             UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification, nil)
         } else {
-            screenshotHelper.takeDelayedScreenshot(tab)
+            // To Screenshot a tab that is hidden we must add the webView,
+            // then wait enough time for the webview to render.
+            if let webView =  tab.webView {
+                view.insertSubview(webView, atIndex: 0)
+                let time = dispatch_time(DISPATCH_TIME_NOW, Int64(500 * NSEC_PER_MSEC))
```

Comment:
I tried using takePendingScreenshot but it didn't work. The only issue with takePendingScreenshot is that it calls the takeScreenshot which then calls "drawViewHierarchyInRect", and that method can only take a screenshot of views that are currently being rendered on the screen.  So if we add a new tab in the background, it's not being rendered so the screenshot will be white until we click on the tab (and the view is rendered).  What this does is just adds the views just long enough to render and take a screen shot, then remove it.